airflow/configuration.py

```diff
@@ -572,63 +588,116 @@ def get(self, section: str, key: str, **kwargs) -> str | None:  # type: ignore[o
         section = str(section).lower()
         key = str(key).lower()
 
-        deprecated_section, deprecated_key, _ = self.deprecated_options.get(
-            (section, key), (None, None, None)
-        )
+        issue_warning = True
+        deprecated_section: str | None
+        deprecated_key: str | None
 
-        option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)
+        # For when we rename whole sections
+        if section in self.deprecated_sections:
+            warnings.warn(
+                f"The config section [{section}] has been renamed to {self.deprecated_sections[section][0]}."
+                " Please update your config.",
+                FutureWarning,
+                stacklevel=2,
```

Comment:
The logging outputs:
```
FutureWarning: The config section [kubernetes_executor] has been renamed to kubernetes. Please update your config.
```
but I think it should be the other way round? i.e renamed to kubernetes_executor instead of renamed to kubernetes

Reply:
Whoops! Good catch

Reply:
Nope, this is correct as it is. Extended tests to ensure it's logged the right way around.